tests/Tests.hs

```diff
@@ -15,11 +15,11 @@
 -- WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 -- License for the specific language governing permissions and limitations
 -- under the License.
---
+
 -- |
--- Module      : Tests
--- Copyright   : (c) 2013-2014 PivotCloud, Inc
--- License     : Apache-2
+-- Module: Main
```

Comment:
Another example why it would be really nice if cabal could be a bit more flexible about names of modules with application entry points. GHC is more flexible. 

However looking at how the discussion on https://github.com/haskell/cabal/issues/172 and it's successor https://github.com/haskell/cabal/issues/1847 went it wonder if it ever will happen...